code-challenges.test.js

```diff
@@ -88,7 +229,18 @@ var vowelTester3 = "challenge"
 // Expected output: 2
 
 
-
-
-
 // b) Create the function that makes the test pass.
+
+//create a function called firstVowel
+//takes in an argument that is a string
+const firstVowel = (str) => {
+    //the string will need to be split in order to give the function the ability to iterate through the indexes
+    var makeStrArr = str.split("");
```

Comment:
descriptive. To clarify with variable and function naming: for variables, you want to describe the data being store. for functions, you want to start with a verb to describe the behavior of the function